src/rp2_common/hardware_pio/pio.c

```diff
@@ -24,7 +24,7 @@ void pio_sm_claim(PIO pio, uint sm) {
     check_sm_param(sm);
     uint which = pio_get_index(pio);
     if (which) {
-        hw_claim_or_assert(&claimed, NUM_PIO_STATE_MACHINES + sm, "PIO 1 SM %d already claimed");
+        hw_claim_or_assert(&claimed, NUM_PIO_STATE_MACHINES + sm, "PIO 1 SM (%d - 4) already claimed");
```

Comment:
So I guess this is to change e.g. "PIO 1 SM 5 already claimed" to ""PIO 1 SM (5 - 4) already claimed"? (with the implication that the user will be able to "understand" that this _actually_ means "PIO 1 SM 1 already claimed"?)